kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/PartitionValueEvaluator.java

```diff
@@ -86,6 +86,9 @@ public long getLong(int rowId) {
                 } else if (partitionType.equivalent(TimestampType.TIMESTAMP)) {
                     return InternalUtils.microsSinceEpoch(
                         Timestamp.valueOf(input.getString(rowId)));
+                } else if (partitionType.equivalent(TimestampNTZType.TIMESTAMP_NTZ)) {
```

Comment:
minor nit: _might_ be safer to do
```java
else if (partitionType.equivalent(TimestampType.TIMESTAMP) ||
        partitionType.equivalent(TimestampNTZType.TIMESTAMP_NTZ))) {
    return InternalUtils.microsSinceEpoch(
        Timestamp.valueOf(input.getString(rowId)));  
}
```

then one implementation won't accidentally diverge from the other in the future